lib/ServerDate.js

```diff
@@ -175,8 +175,9 @@ function synchronize() {
   function requestSample() {
     var request = new XMLHttpRequest();
 
-    // Ask the server for its opinion of the current time (milliseconds).
-    request.open("GET", URL + "?time=now");
+    // Ask the server for another copy of ServerDate.js but specify a unique number on the URL querystring
+	// so that we don't get the browser cached Javascript file
+	request.open("GET", URL + "?time=" + (new Date()).getTime.toString());
```

Comment:
First let's use a different query field name because `time` is misleading in this context.  How about `noCache`?

Second, I think you probably want this instead:

```JS
"?noCache=" + Date.now()
```